app/main.py

```diff
@@ -1 +1,50 @@
-#  write your code here
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        upd_appetite = self.appetite
+        if self.is_hungry is True:
```

Comment:
There is no need to explicitly check that the boolean `True` is stored there. Remove the `is True` part from your if condition.

Reply:
This comment was not implemented. Additional comment is left.